apps/web/src/pages/DirectiveForms/DigitalServicesContractingQuestionnaire/useLabels.ts

```diff
@@ -0,0 +1,533 @@
+    }),
+    hasImmediateImpactOnOtherDepartments: intl.formatMessage({
+      defaultMessage:
+        "Do you expect any potential immediate carry-forward / ripple effect on other departments in terms of staffing levels or skill sets required?",
+      id: "gsaza3",
+      description:
+        "Label for _has immediate impact on other departments_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    hasFutureImpactOnOtherDepartments: intl.formatMessage({
+      defaultMessage:
+        "Do you expect any potential long-term carry-forward / ripple effect on other departments in terms of staffing levels or skill sets required?",
+      id: "0aU6BD",
+      description:
+        "Label for _has future impact on other departments_ fieldset in the _digital services contracting questionnaire_",
+    }),
+
+    // Operations considerations
+    hasOperationsConsiderations: intl.formatMessage({
+      defaultMessage:
+        "Do any of the listed operational factors influence the decision to contract?",
+      id: "NDkViv",
+      description:
+        "Label for _has operations considerations_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    operationsConsiderations: intl.formatMessage({
+      defaultMessage:
+        "Select all the factors that have influenced the decision to contract.",
+      id: "E3lQMA",
+      description:
+        "Label for _operations considerations_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    operationsConsiderationsOther: intl.formatMessage({
+      defaultMessage: "Please specify the factor",
+      id: "1MtuHa",
+      description:
+        "Label for _other operations considerations_ fieldset in the _digital services contracting questionnaire_",
+    }),
+
+    // Talent sourcing decision
+    contractingRationalePrimary: intl.formatMessage({
+      defaultMessage: "Select the primary rationale",
+      id: "dwFVEN",
+      description:
+        "Label for _primary contracting rationale_ fieldset in the _digital services contracting questionnaire_",
+    }),
+    contractingRationalePrimaryOther: intl.formatMessage({
+      defaultMessage: "Please specify the other primary rationale",
+      id: "VYzixZ",
+      description:
+        "Label for _an other contracting rationale_ field in the _digital services contracting questionnaire_",
+    }),
+    ocioConfirmedTalentShortage: intl.formatMessage(
+      {
+        defaultMessage:
+          "Has OCIO confirmed that there is no available pre-qualified talent in an <link>OCIO-coordinated talent pool</link> that could meet the need in the timeframe provided?",
+        id: "yGW8Y5",
+        description:
+          "Label for _OCIO confirmed talent shortage_ field in the _digital services contracting questionnaire_",
+      },
+      {
+        link: (chunks: React.ReactNode) =>
+          buildExternalLink(paths.search(), chunks),
+      },
```

Comment:
Same here, I'm not a huge fan of including links in field labels. Clicking a label should focus the input, but this breaks that pattern for part of it.

Reply:
Response from Rosita: link already exists above - we can remove this one

Reply:
[Remove link for label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7771/commits/b32c5bbebac92f29a08743e99acd2093436a3f5f)